packages/addons/paraglide/index.ts

```diff
@@ -271,11 +208,15 @@ export default defineAddon({
 				const links = validLanguageTags
 					.map(
 						(x) =>
-							`${templateCode.getIndentString()}<button onclick={() => switchToLanguage('${x}')}>${x}</button>`
+							`${templateCode.getIndentString()}<button onclick={() => setLocale('${x}')}>${x}</button>`
 					)
 					.join('\n');
 				templateCode.append(`<div>\n${links}\n</div>`);
 
+				templateCode.append(
+					'<p>\nIf you use VSCode, install the <a href="https://marketplace.visualstudio.com/items?itemName=inlang.vs-code-extension" target="_blank">Sherlock i18n extension</a> for a better i18n experience.\n</p>'
+				);
+
```

Comment:
Moved the recommendation form the CLI into the paraglide js demo with a link to directly open the vscode marketplace